src/debugging/DockerDebugConfigurationProvider.ts

```diff
@@ -24,7 +24,8 @@ export class DockerDebugConfigurationProvider implements DebugConfigurationProvi
     ) { }
 
     public provideDebugConfigurations(folder: WorkspaceFolder | undefined, token?: CancellationToken): ProviderResult<DebugConfiguration[]> {
-        return undefined;
+        window.showErrorMessage("To debug in a Docker container on supported platforms, use the command \"Docker: Add Docker Files to Workspace\".");

```

Comment:
Couldn't they also use the "initialize for debugging" command, too?

Reply:
Should we show an error message with button(s) to immediately run either of those commands?

Reply:
They could do both, but I'm guessing that most people running into this situation have not added Docker support in the first place. For future projects, if they had, it would already have the debug configuration and they wouldn't hit this.

The Add workflow does ask before overwriting, so IMO it's safe.

We can add a button, shouldn't be difficult.